docs/2023/BeckN/BeckN Smart Policy Infrastructure/updates/2023-07-28.md

```diff
@@ -11,6 +11,61 @@ author: Aakanksha Jaiswar
 
 ## Screenshots / Videos 
 
-## Contributions
+a. *Personal Information:*
+   - Name
+   - Email Address
+   - Contact Number
+   - Date of Birth
+   - Gender
+
+b. *Account Credentials:*
+   - Username
+   - Password (with validation criteria)
+   - Security Question and Answer (for password recovery)
+
+c. *Address Details:*
+   - Billing Address
+   - Shipping Address (if different from billing address)
+
+d. *Company Information:*
+   - Company Name (if applicable)
+   - Job Title (if applicable)
+
+e. *Consent and Communication Preferences:*
+   - Consent to Terms of Service and Privacy Policy
+   - Subscription Preferences (e.g., newsletters, promotions)
+
+*4. User Order History Building Blocks:*
+For maintaining user order history, the following fields were found to be essential:
+
+a. *Order Information:*
+   - Order ID
+   - Order Date
+   - Order Total
 
+b. *Product Details:*
+   - Product ID
+   - Product Name
+   - Quantity
+   - Price
+
+c. *Payment Information:*
+   - Payment Method (e.g., credit card, PayPal)
+   - Payment Status
+
+d. *Shipping Information:*
+   - Shipping Method
+   - Shipping Address
+   - Tracking Number (if applicable)
```

Comment:
I think it is irrelevant here. You should come up with an optimal way to share your milestones, contributions and screenshots/videos.